server/src-lib/Hasura/RQL/DDL/Remote/Validate.hs

```diff
@@ -12,31 +12,31 @@ module Hasura.RQL.DDL.Remote.Validate
   ) where
 
 import           Data.Bifunctor
-import           Data.HashMap.Strict (HashMap)
-import qualified Data.HashMap.Strict as HM
-import           Data.List.NonEmpty (NonEmpty(..))
+import           Data.List.NonEmpty                (NonEmpty (..))
 import           Data.Validation
-import qualified Hasura.GraphQL.Context as GC
-import qualified Hasura.GraphQL.Schema as GS
 import           Hasura.GraphQL.Validate.Types
-import qualified Hasura.GraphQL.Validate.Types as VT
 import           Hasura.Prelude
 import           Hasura.RQL.DDL.Relationship.Types
 import           Hasura.RQL.DDL.Remote.Input
 import           Hasura.RQL.Types
 import           Hasura.SQL.Types
-import qualified Language.GraphQL.Draft.Syntax as G
 
+import qualified Data.HashMap.Strict               as HM
+import qualified Data.Text                         as T
+import qualified Hasura.GraphQL.Context            as GC
+import qualified Hasura.GraphQL.Schema             as GS
+import qualified Language.GraphQL.Draft.Syntax     as G
```

Comment:
Side comment:

I assume this is stylish-haskell doing this; it creates a "dependency" on the longest import name. If you import a longer name or remove [this one](https://github.com/tirumaraiselvan/graphql-engine/blob/533c3ccb9c441e222be4cdf971de323724a1f315/server/src-lib/Hasura/RQL/DDL/Remote/Validate.hs#L19) then the whole import list gets re-indented like it did here, creating more changes in the diff. Not a big deal in this example, but it creates more merge conflicts and nobody likes merge conflicts.

The `hindent` tool also avoids any "dependency", by line-breaking often. Renaming something and reformatting should never change more lines of code (like it would [here](https://github.com/tirumaraiselvan/graphql-engine/blob/533c3ccb9c441e222be4cdf971de323724a1f315/server/src-lib/Hasura/RQL/DDL/Remote/Validate.hs#L157)) and make diffs/merging more complicated.

Reply:
I did accidentally [commit changes to an import list here](https://github.com/tirumaraiselvan/graphql-engine/commit/0e5931fb99fef796d386a50faa38c385c06004f2#diff-f3e9f4f650156ebb1e73759287ff254dL29), but normally I try not to make formatting changes to existing code. Sorry about that.

Reply:
How do you recommend not doing these no-op changes while keeping stylish-haskell enabled?

Reply:
I invoke formatting tools on a key rather than having them automatic.